ansible-playbook.yml

```diff
@@ -0,0 +1,132 @@
+---
+- name: Install To-Do App on new web server
+  hosts: managednodes
+  remote_user: ec2-user
+  vars_prompt:
+    - name: trello_api_key
+      prompt: Trello API key?
+      default: "1"
```

Comment:
I know we were using these to speed up testing, but it would be nice to remove them now! We could set the Board/List ids to their actual defaults if we wanted, and just leave the Trello credentials to be inputted